model/src/main/java/com/basistech/rosette/apimodel/recordsimilarity/RecordSimilarityResponse.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2024 Basis Technology Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.basistech.rosette.apimodel.recordsimilarity;
+
+import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Builder;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+import com.basistech.rosette.apimodel.Response;
+
+/**
+ * Response data model for comparison of two records.
+ */
+@Value
+@Builder
+@Jacksonized
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class RecordSimilarityResponse extends Response {
+    Map<String, RecordSimilarityFieldInfo> fields;
```

Comment:
Sonar says you need "@Valid" here. 
https://sonar.basistech.net/project/issues?id=com.basistech.rosette%3Arosette-api-java-binding&open=AY5SZyAcDn-JuiNfWcaa&pullRequest=215&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL